frontend/include/chpl/util/version-info.h

```diff
@@ -58,10 +58,10 @@ const char* getConfiguredPrefix();
 std::string getMajorMinorVersion();
 
 /*
-  get the chpl version string. Developer flag helps decide if a git-sha is
-  included or not
+  get the chpl version string. release versions will have format Major.Minor.Patch or
```

Comment:
While here, could you capitalize the first word of these sentences? `Get` and `Release`.